lib/formtastic_customizations.rb

```diff
@@ -17,9 +17,15 @@ def hint_html
           
           output = "".html_safe
           output << hint_html_without_customizations
-                    
+
+          current_person = template.try(:current_user).try(:person_type)
+          if current_person.nil?
```

Comment:
Agreed. Also, I would recommend that we move the `if..then` logic to the `HelpText` model instead of doing it here in the view. Recommend changing lines 21–26 to:

```
audience = template.try(:current_user).try(:person_type)
help_text = HelpText.for(object_name.to_s.classify, method, audience)
```

Same with down below.